tests/features/ecas-login.feature

```diff
@@ -34,3 +34,30 @@ Feature: Login through OE Authentication
     And I should not see the link "My account"
     And I should not see the link "Log out"
     And I should see the link "Log in"
+
+  Scenario: A blocked user should not be able to log in
+    When I am on the homepage
```

Comment:
Is the first part of the scenario needed? We are testing the successful login in the previous scenario no?